src/chainparams.cpp

```diff
@@ -163,13 +163,13 @@ class CTestNetParams : public CMainParams {
         nMaxTipAge = 0x7fffffff;
         nPruneAfterHeight = 1000;
 
-        // 1MB max blocks before 1 Aug 2015
-        // Then, if miner consensus: 8MB max, doubling every two years
+        // 1MB max blocks before 1 Mar 2014
+        // Then, if miner consensus: 2MB max, doubling twice over two years
         consensus.nMaxSizePreFork = 1000*1000; // 1MB max pre-fork
-        consensus.nEarliestSizeForkTime = 1438387200; // 1 Aug 2015 00:00:00 UTC
+        consensus.nEarliestSizeForkTime = 1393632000; // 01 Mar 2014 00:00:00 UTC, 2 years ahead of mainnet
```

Comment:
FWIW the reasoning is explicitly stated in the [comments](https://github.com/bitcoinclassic/bitcoinclassic/blob/b0bb4ebb7f39de5f4e68cac445dc65ced3b02b26/src/consensus/params.h#L46-L49) inside `Consensus::Params::MaxBlockSize()`:

```
        // Piecewise-linear-between-doublings growth. Calculated based on a fixed                                                                                                                   
        // timestamp and not the activation time so the maximum size is                                                                                                                             
        // predictable, and so the activation time can be completely removed in                                                                                                                     
        // a future version of this code after the fork is complete.  
```

The artifact you mention seems reasonable to me given the above justification but IMO this case should be made clear to miners so as to not give them _any_ surprise which could potentially violate their trust.